.jshintrc

```diff
@@ -13,5 +13,6 @@
   "node": true,
   "quotmark": true,
   "camelcase": true,
-  "maxlen": 80
+  "maxlen": 80,
+  "nonbsp": true
```

Comment:
Why was this added? I ran grunt jshint without it and did not receive any errors. Based on the describe of that jshint option I'm not sure why you would need it.